src/tss2-sys/api/Tss2_Sys_ACT_SetTimeout.c

```diff
@@ -43,7 +48,9 @@ TSS2_RC Tss2_Sys_ACT_SetTimeout_Prepare(
 
     return CommonPrepareEpilogue(ctx);
 }
+#endif
 
+#if !defined(CONFIGURATOR) || defined(ENABLE_TSS2_SYS_ACT_SETTIMEOUT_COMPLETE)
```

Comment:
I think I'd be more happy with a
```
#if !defined(DISABLE_TSS2_SYS_ACT_SETTIMEOUT_COMPLETE)
```
and similarly for the rest.

The configurator.h could include said macros then.